uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -688,6 +688,43 @@ private function getEntitlementCalculationMock(
     return $calculation;
   }
 
```

Comment:
You've added 3 new methods but you've added tests to only one of them. Please add test coverage to all the methods